pkg/runner/runner.go

```diff
@@ -340,9 +341,18 @@ func (r *Runner) handleCredentials(callCtx engine.Context, monitor Monitor, env
 	}
 
 	for _, credToolName := range callCtx.Tool.Credentials {
-		cred, exists, err := store.Get(credToolName)
-		if err != nil {
-			return nil, fmt.Errorf("failed to get credentials for tool %s: %w", credToolName, err)
+		var (
+			cred   *credentials.Credential
+			exists bool
+			err    error
+		)
+
+		// Only try to look up the cred if the tool is on GitHub.
+		if strings.HasPrefix(credToolName, "github.com") {
```

Comment:
nit: I suggest using a constant instead of a literal since we're using it in multiple places.

Reply:
Good call

Reply:
Fixed. Decided to just define a function.